core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -979,6 +979,9 @@ private void listStatusInternal(LockedInodePath currInodePath, AuditContext audi
             InodeTree.LockMode.READ, child, childComponents)) {
           listStatusInternal(childInodePath, auditContext,
               nextDescendantType, statusList);
+        } catch (InvalidPathException e) {
+          LOG.warn("an invalid ptah was list, parent directory {}",
```

Comment:
Can we move this to `debug` level? More information about Alluxio logging conventions: https://www.alluxio.org/docs/master/en/contributor/Code-Conventions.html#logging-conventions